### PR TITLE
Rest Service

### PR DESCRIPTION
**Pull request by arcturus2**
### Description of the Change

#279

### CODE REVIEW

File: CoreWebServer/src/au/gov/asd/tac/constellation/webserver/WebServer.java [hunk 1]
```diff
@@ -180,7 +179,6 @@ public static synchronized int start() {
                     try {
                         server.join();
                     } catch (InterruptedException e) {
-                        Thread.currentThread().interrupt();
```
Comment (antares1470) [inline CoreWebServer/src/au/gov/asd/tac/constellation/webserver/WebServer.java]: Was this meant to be removed?

File: CoreWebServer/src/au/gov/asd/tac/constellation/webserver/WebServer.java [hunk 2]
```diff
@@ -120,7 +120,6 @@ public static boolean checkSecret(final HttpServletRequest request, final HttpSe
     private static int port = 0;
 
     static final String CONSTELLATION_CLIENT = "constellation_client.py";
-    static final String RESOURCES = "resources/";
```
Comment (antares1470) [inline CoreWebServer/src/au/gov/asd/tac/constellation/webserver/WebServer.java]: Is there a reason why a constant isn't being used anymore?

File: CoreWebServer/src/au/gov/asd/tac/constellation/webserver/WebServer.java (hunk 1 above)
Comment (arcturus2) [reply CoreWebServer/src/au/gov/asd/tac/constellation/webserver/WebServer.java]: nope I will have it added back in.

File: CoreWebServer/src/au/gov/asd/tac/constellation/webserver/WebServer.java (hunk 2 above)
Comment (arcturus2) [reply CoreWebServer/src/au/gov/asd/tac/constellation/webserver/WebServer.java]: @antares1470 i think the issue was dealing with merge conflicts and some of your changes being accidentally overridden.

**Pull request merged**